src/plugins/vis_type_timeseries/public/application/components/color_picker.test.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { ColorPicker, ColorPickerProps } from './color_picker';
+import { mount } from 'enzyme';
+import { ReactWrapper } from 'enzyme';
+import { EuiColorPicker, EuiIconTip } from '@elastic/eui';
+
+describe('ColorPicker', () => {
+  let defaultProps: ColorPickerProps;
```

Comment:
nitpick: instead of creating `beforeAll` to assign `defaultProps` to value, you could simply do (replacing 27-38 lines)
```
  const defaultProps: ColorPickerProps = {
    name: 'color',
    value: null,
    onChange: jest.fn(),
    disableTrash: true,
  };
let component: ReactWrapper<ColorPickerProps>;
```

Reply:
You are right :)